client/src/components/Shared/AverageStars.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import helper from './helpers.js'
+import FullStar from './../Review Widget/icons/star.svg'
+import HalfStar from './../Review Widget/icons/star-half.svg'
+import QuarterStar from './../Review Widget/icons/star-one-quarter.svg'
+import ThreeQuarterStar from './../Review Widget/icons/star-three-quarter.svg'
+import EmptyStar from './../Review Widget/icons/empty-star.svg'
+
+const AverageStars = (props) => {
+  const averageRating = helper.findStarRating(props.rating);
+
+  let wholeRating = Math.floor(averageRating) || 0;
```

Comment:
Was running into array length issues until I changed let wholeRating = Math.floor(averageRating) into an OR operator Math.floor(averageRating) || 0. Probably needed to do this in order to account falsey values for undefined, null, and NaN. May be redundant since I refactored my findStarRatings in helpers.js...but it works?